app/_components/cbio-util.js

```diff
@@ -666,6 +672,47 @@ cbio.util = (function() {
     return def.promise();
   }
 
+
+  function getDecimalExponents(data){
+
+    // Copy the values, rather than operating on references to existing values
+    if (data === null || data.length < 1) {//if data is null or empty, return data
+      return data;
+    }
+    
+    var values = [];
+    _.each(data, function(item) {
+      if (!isNaN(item)) {
+        values.push(Number(item));
+      }
+    });
+
+    // Then sort
+    values.sort(function(a, b) {
```

Comment:
I didn't explain clearly. Values.sort() in here return data based on kind of alphabetical order, so I have to mention I want to sort data in ascending order.